ll_kth_from_end.js

```diff
@@ -0,0 +1,49 @@
+class LinkedList {
+    constructor() {
+      this.head = null;
+      this.length = 0;
+    }
+   
+    add(value) {
+      var node = new Node(value);
+      if (this.head == null) {
+        this.head = node;
+        this.length++;
+      } else {
+        var current = this.head;
+        while (current.next) {
+          current = current.next;
+        }
+        current.next = node;
+        this.length++;
+      }
+    }
+   
+    reverse(k) {
```

Comment:
If this is the code you are using to "find kth from end" you might want to rename this method so that its functionality is clear!